specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2024-06-02-preview/managedClusters.json

```diff
@@ -10126,4 +10167,4 @@
       "x-ms-parameter-location": "method"
     }
   }
-}
+}
```

Comment:
I guess change on this caused the failure of the Swagger PrettierCheck.

Reply:
gotta love vscode auto formatting 😄 ran prettier + pushed!